app/controllers/account_reset/confirm_request_controller.rb

```diff
@@ -5,10 +5,11 @@ def show
       if email.blank?
         redirect_to root_url
       else
-        render :show, locals: {
-          email: email,
-          sms_phone: TwoFactorAuthentication::PhonePolicy.new(current_user).configured?,
-        }
+        render :show,
+               locals: {
+                 email: email,
+                 sms_phone: TwoFactorAuthentication::PhonePolicy.new(current_user).configured?,
```

Comment:
There are cases where this style of indenting can make some lines way too long (I know because I haven't figured out how to get RubyMine to stop doing this)